paddlers/transforms/operators.py

```diff
@@ -1959,15 +1960,28 @@ class AppendIndex(Transform):
         index_type (str): Type of remote sensinng index. See supported 
             index types in 
             https://github.com/PaddlePaddle/PaddleRS/tree/develop/paddlers/transforms/indices.py .
-        band_indices (dict): Mapping of band names to band indices 
+        band_indices (dict, optional): Mapping of band names to band indices 
             (starting from 1). See band names in 
-            https://github.com/PaddlePaddle/PaddleRS/tree/develop/paddlers/transforms/indices.py . 
+            https://github.com/PaddlePaddle/PaddleRS/tree/develop/paddlers/transforms/indices.py .
+            Default: None.
+        sensor (str, optional): Type of sensor, used to determine the description of each band.
```

Comment:
`Type of sensor, used to determine the description of each band.` -> `Type of satellite. If set, band indices will be automatically determined accordingly.`